src/libraries/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/AsnEncodedData.cs

```diff
@@ -12,6 +13,7 @@ public class AsnEncodedData
     {
         protected AsnEncodedData()
         {
+            _rawData = Array.Empty<byte>();
```

Comment:
@bartonjs, what we discussed offline.

Reply:
@bartonjs, a handful of Pkcs tests are expecting RawData to be null, e.g.
https://github.com/dotnet/runtime/blob/ee0aadc3dad5c8d64ce67c6f7ddeb07326aeb8ec/src/libraries/System.Security.Cryptography.Pkcs/tests/Pkcs9AttributeTests.cs#L14-L20
Would you prefer:
a) I revert these changes and make `RawData` return `byte[]?` instead of `byte[]`, or
b) Update the tests to not require `null`
?

Reply:
If there's actually places that we were letting the `null` leak out before, then (a).  Sorry for leading things astray here.

Reply:
I separated this out as https://github.com/dotnet/runtime/pull/37572.